src/main/java/nextstep/subway/domain/Section.java

```diff
@@ -0,0 +1,146 @@
+package nextstep.subway.domain;
+
+import java.util.Objects;
+
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+
+@Entity
+public class Section extends BaseEntity {
+	private static final int MIN_DISTANCE = 0;
+	private static final int INIT_SECTION_ORDER = 0;
+
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	private Long id;
+
+	@ManyToOne(fetch = FetchType.LAZY)
+	@JoinColumn(name = "up_station_id")
```

Comment:
사소하지만 구간에 상행역과 하행역은 필수이기에 명시적으로 필요하다는 것을 알려줘도 좋을 것 같아요 :)
```suggestion
	@ManyToOne(optional = false, fetch = FetchType.LAZY)
        @JoinColumn(name = "up_station_id", nullable = false)
```

Reply:
맞습니다 !! 추가하겠습니다!